### ISSUE TITLE
`case` blocks get completely broken

### ISSUE THREAD

**Issue by wrboyce** [Loilo/node-bash-minifier#2]
```sh
case "$var" in
	a)
		echo 'var=a!'
		;;
	b)
		echo 'var=b!'
		;;
	*)
		echo "var=*!"
		;;
esac
```

becomes

```sh
case "$var" in a);echo 'var=a!';;;b);echo 'var=b!';;;*);echo "var=*!";;;esac
```

You have to write the `case` in a very specific manner for it to continue to work after minification

```sh
case "$var" in
	a) 	echo 'var=a!' ;;
	b) 	echo 'var=b!' ;;
	*) 	echo "var=*!" ;;
esac
```

The first line must be on the same line as the condition (to avoid a comma after the `)`), and the last line (`;;`) must be on the line above as to avoid a triple-semicolon.

This will produce syntactically valid code:

```sh
case "$var" in a)echo 'var=a!';;b)echo 'var=b!';;*)echo "var=*!";;esac
```

**Comment by loilo** [comment]
Hey, thanks for the report.

Unfortunately, there's not much I can do here. As mentioned [in the Readme](https://github.com/Loilo/node-bash-minifier#contributing), this project is a strict port of [its Python equivalent](https://github.com/precious/bash_minifier) (which doesn't seem to be maintained anymore, at least there haven't been any reactions to issues in more than a year).

Since it's a strict port, I'm not familiar with the logic and the inner workings of the minifier. In general, its whole approach seems quite error-prone to me today because it basically does some glorified search&replace instead of proper parsing.

**Comment by wrboyce** [comment]
Fair enough, at least the issue exists with a workaround now for anyone else who may have this issue.

**Comment by loilo** [comment]
FWIW, since the original minifier is unmaintained, I don't feel committed to retain compatibility with it.
So, if you want to dig into the source code, feel free to attach a PR with your fix here.
